modules/internal/ChapelBase.chpl

```diff
@@ -1496,8 +1496,11 @@ module ChapelBase {
   inline proc >(a: uint(64), param b: uint(64)) {
     return __primitive(">", a, b);
   }
+  inline proc >(param a: uint(?w), b: uint(w)) param where a == 0 {
```

Comment:
Seems a little inconsistent  that you use `uint(?w)` while the other versions use `uint(64)` and rely on coercion. Is there any reason to not keep the signatures consistent?